src/docs/components/cursor.md

```diff
@@ -6,7 +6,7 @@ parent_section: components
 order: 2
 ---
 
-The cursor component defines interaction with the scene through clicking and gazing, by using a raycaster to determine which object has been clicked. When the mouse is clicked, the closest visible entity intersecting the cursor will have a DOM click event triggered.
+The `cursor` component defines interaction with the scene through clicking and gazing, by using a [raycaster](https://en.wikipedia.org/wiki/Ray_casting) to determine which object has been clicked. When the mouse is clicked, the closest visible entity intersecting the cursor will have a DOM `click` event triggered.
```

Comment:
ah, actually uses `dependencies` - gotcha. I thought the raycaster code was just shared